models/userDao.js

```diff
@@ -0,0 +1,69 @@
+const { appDataSource } = require("./dbconfig");
+
+const getHashedPassword = async (email) => {
+  try {
+    const hashedPassword = await appDataSource.query(
+      `SELECT
+                password AS hashedPassword
+            FROM
+                users
+            WHERE
+                email = ?;
+            `,
+      [email]
+    );
+    return hashedPassword;
```

Comment:
```suggestion
    return appDataSource.query(
      `SELECT
                password AS hashedPassword
            FROM
                users
            WHERE
                email = ?;
            `,
      [email]
    );
```
작성해주신대로라면 변수 선언 없이 바로 return되어도 됩니다.